Userland/Libraries/LibWeb/MimeSniff/MimeType.cpp

```diff
@@ -22,11 +22,11 @@ namespace Web::MimeSniff {
 bool is_javascript_mime_type_essence_match(StringView string)
 {
     // A string is a JavaScript MIME type essence match if it is an ASCII case-insensitive match for one of the JavaScript MIME type essence strings.
-    // NOTE: The mime type parser automatically lowercases the essence.
-    auto type = MimeType::parse(string);
-    if (!type.has_value())
-        return false;
-    return type->is_javascript();
+    for (const auto& javascript_essence : javascript_mime_type_essence_strings) {
```

Comment:
I could not find a better way to extract this list without creating new functions that accept containers on StringView.h
Any ideas?

Reply:
Using for loops like this is pretty common elsewhere. I think this is fine.